hedera-node/configuration/mainnet/settings.txt

```diff
@@ -1,32 +1,47 @@
-checkSignedStateFromDisk,                      1
-csvFileName,                                   MainNetStats
-csvOutputFolder,                               data/stats
-dbBackup.active,                               1
-dbConnection.active,                           1
-dbRestore.active,                              1
+                                          #############################
+                                          #    Gossip / Networking    #
+                                          #############################
+
+chatter.useChatter,                            false
 doUpnp,                                        false
-enableEventStreaming,                          true
-eventsLogDir,                                  /opt/hgcapp/eventsStreams
-eventsLogPeriod,                               5
-lockLogTimeout,                                4000
-maxEventQueueForCons,                          1000
 maxOutgoingSyncs,                              8
 numConnections,                                1000
-reconnect.active,                              1
-reconnect.asyncStreamTimeoutMilliseconds,      60000
-reconnect.reconnectWindowSeconds,              -1
-showDbStats,                                   0
-showInternalStats,                             1
-state.saveStatePeriod,                         900
-state.signedStateDisk,                         5
-state.signedStateKeep,                         10
-throttle7extra,                                0.5
 useLoopbackIp,                                 false
-waitAtStartup,                                 false
+
+                                          #############################
+                                          #           State           #
+                                          #############################
+
 jasperDb.iteratorInputBufferBytes,             16777216
 merkleDb.iteratorInputBufferBytes,             16777216
-virtualMap.preferredFlushQueueSize,            10000
+state.checkSignedStateFromDisk,                true
+state.saveStatePeriod,                         900
+state.signedStateDisk,                         5
+state.signedStateKeep,                         10
 state.mainClassNameOverride,                   com.hedera.services.ServicesMain
-prometheusEndpointEnabled,                     true
-chatter.useChatter,                            false
-transThrottle,                                 false
\ No newline at end of file
+virtualMap.preferredFlushQueueSize,            10000
+
+                                          #############################
+                                          #          Events           #
+                                          #############################
+
+event.enableEventStreaming,                    true
```

Comment:
The default in the code is `false`. It should be changed to `true`, but in a different PR. There are probably several defaults that should change so that they are the value for mainnet.

Reply:
I agree. It makes a lot of sense to me to push default settings values towards what we use on mainnet if we know it is a setting we will always want on mainnet. Another example is `reconnect.active`. Tracked by this ticket: https://github.com/hashgraph/hedera-services/issues/7206